### ISSUE TITLE
Missing styles.css file in version 0.4.27

### ISSUE THREAD

**Issue by t0oF-azpn** [emerson-eps/color-tables#338]
When updating our project, it failed to compile:
```
Failed to compile.

Module not found: Error: Can't resolve '../../css/styles.css' in 'G:\dev\AzureDevOps\dashboard-frontend\node_modules\@emerson-eps\color-tables\dist\component\ColorSelector'
```
Investigating, this file was not added to the package.

Furthermore, you should consider to use an approach like (to be investigated):
```
import "!style-loader!css-loader!../../css/styles.css";
```
to insure it still works fine in client applications.

**Comment by shruthirai** [comment]
@t0oF-azpn Yes, we noticed this issue as well, will fix it soon, thanks.

**Comment by t0oF-azpn** [comment]
Fixed with 0.4.29 
Thank you
